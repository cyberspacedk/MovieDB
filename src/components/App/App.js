import React from 'react';
import 'antd/dist/antd.css';
// styles
import GlobalStyle from '../../globalStyles';
import '../../assets/styles/app.scss';
import Home from '../Home';

function App() {
  return (
    <div className="App">
      <Home />
      <GlobalStyle />
    </div>
  );
}

export default App;
