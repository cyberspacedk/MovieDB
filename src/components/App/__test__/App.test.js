import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from '../App';

describe('Component: App', () => {
  const AppComponent = shallow(<App />);
  it('should match its snapshot', () => {
    expect(shallowToJson(AppComponent)).toMatchSnapshot();
  });
});
