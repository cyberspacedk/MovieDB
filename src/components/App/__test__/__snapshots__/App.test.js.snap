// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its snapshot 1`] = `
<div
  className="App"
>
  <Connect(ParentForm) />
  <Connect(TopFilmsContainer) />
  <GlobalStyleComponent />
</div>
`;
